TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java

```diff
@@ -156,7 +157,7 @@ public static void start(CommandContext cmd, CommandSender sender) {
                     sender.sendMessage(ChatColor.RED + "Unknown time \"" + cmd.getString(0) + "\"");
                 }
             }
-          sender.sendMessage(ChatColor.GREEN + "Match will start in " + time + " seconds.");
+          sender.sendMessage(ChatColor.GREEN + "Match will start in " + time + " second" + (String.valueOf(time).endsWith("1") ? "" : "s") + ".");
```

Comment:
One more problem: This will make it say "21 second", "11 second", etc. You should just check if time == 1. I like the idea though.